tests/tx/test_tx_storage.py

```diff
@@ -162,6 +162,29 @@ def test_storage_basic_v2(self):
         self.assertEqual(2, len(tx_parents_hash))
         self.assertEqual(set(tx_parents_hash), {self.genesis_txs[0].hash, self.genesis_txs[1].hash})
 
+    def test_vertices_count(self):
+        _set_test_mode(TestMode.TEST_ALL_WEIGHT)
+
+        _new_blocks_count = 0
```

Comment:
Why don't you rename it to `blocks_count` and initializes it with `1`? This way you can remove the hard `+ 1` at the asserts. You can do the same with `_new_txs_count`.

Reply:
Done!